src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
+    String blabla;
 
     public ReviewResponse(int line, String comment) {
```

Comment:
:warning: Using '==' for String comparison can lead to unexpected behavior. Should use equals() method for String comparison. (Severity: Major)